src/endpoint/iam/iam_utils.js

```diff
@@ -82,6 +82,34 @@ function check_iam_path_was_set(iam_path) {
     return iam_path && iam_path !== IAM_DEFAULT_PATH;
 }
 
+/**
+ * parse_max_items converts the input to the needed type
+ * @param {any} input_max_items
+ */
+function parse_max_items(input_max_items) {
+    if (_.isUndefined(input_max_items)) return;
```

Comment:
no need for this function, just compare with x===undefined

Reply:
Changed, thanks.